core/modules/replica/sql/replication_config_ncsa.sql

```diff
@@ -47,36 +47,36 @@ INSERT INTO `config` VALUES ('worker', 'db_password',                'CHANGEME')
-INSERT INTO `config_worker` VALUES ('db13', 1, 0, 'lsst-qserv-db13',  NULL, 'lsst-qserv-db13',  NULL, NULL, 'lsst-qserv-db13', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db14', 1, 0, 'lsst-qserv-db14',  NULL, 'lsst-qserv-db14',  NULL, NULL, 'lsst-qserv-db14', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db15', 1, 0, 'lsst-qserv-db15',  NULL, 'lsst-qserv-db15',  NULL, NULL, 'lsst-qserv-db15', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db16', 1, 0, 'lsst-qserv-db16',  NULL, 'lsst-qserv-db16',  NULL, NULL, 'lsst-qserv-db16', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db17', 1, 0, 'lsst-qserv-db17',  NULL, 'lsst-qserv-db17',  NULL, NULL, 'lsst-qserv-db17', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db18', 1, 0, 'lsst-qserv-db18',  NULL, 'lsst-qserv-db18',  NULL, NULL, 'lsst-qserv-db18', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db19', 1, 0, 'lsst-qserv-db19',  NULL, 'lsst-qserv-db19',  NULL, NULL, 'lsst-qserv-db19', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db20', 1, 0, 'lsst-qserv-db20',  NULL, 'lsst-qserv-db20',  NULL, NULL, 'lsst-qserv-db20', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db21', 1, 0, 'lsst-qserv-db21',  NULL, 'lsst-qserv-db21',  NULL, NULL, 'lsst-qserv-db21', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db22', 1, 0, 'lsst-qserv-db22',  NULL, 'lsst-qserv-db22',  NULL, NULL, 'lsst-qserv-db22', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db23', 1, 0, 'lsst-qserv-db23',  NULL, 'lsst-qserv-db23',  NULL, NULL, 'lsst-qserv-db23', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db24', 1, 0, 'lsst-qserv-db24',  NULL, 'lsst-qserv-db24',  NULL, NULL, 'lsst-qserv-db24', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db25', 1, 0, 'lsst-qserv-db25',  NULL, 'lsst-qserv-db25',  NULL, NULL, 'lsst-qserv-db25', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db26', 1, 0, 'lsst-qserv-db26',  NULL, 'lsst-qserv-db26',  NULL, NULL, 'lsst-qserv-db26', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db27', 1, 0, 'lsst-qserv-db27',  NULL, 'lsst-qserv-db27',  NULL, NULL, 'lsst-qserv-db27', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db28', 1, 0, 'lsst-qserv-db28',  NULL, 'lsst-qserv-db28',  NULL, NULL, 'lsst-qserv-db28', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db29', 1, 0, 'lsst-qserv-db29',  NULL, 'lsst-qserv-db29',  NULL, NULL, 'lsst-qserv-db29', NULL, NULL, NULL);
-INSERT INTO `config_worker` VALUES ('db30', 1, 0, 'lsst-qserv-db30',  NULL, 'lsst-qserv-db30',  NULL, NULL, 'lsst-qserv-db30', NULL, NULL, NULL);
+INSERT INTO `config_worker` VALUES ('db01', 1, 0, 'lsst-qserv-db01',  NULL, 'lsst-qserv-db01',  NULL, NULL, '127.0.0.1', NULL, NULL, NULL);
+INSERT INTO `config_worker` VALUES ('db02', 1, 0, 'lsst-qserv-db02',  NULL, 'lsst-qserv-db02',  NULL, NULL, '127.0.0.1', NULL, NULL, NULL);
```

Comment:
just making sure you want to change to localhost. maybe you want to define a constant for that? Probably not necessary...

Reply:
This is within an SQL DML file. And this value is forced by the DDL file `replica/sql/replication.sql` where the filed `db_host` is defined as `NOT NULL `:
```
CREATE TABLE IF NOT EXISTS `config_worker` (

  `name`         VARCHAR(255)       NOT NULL ,      -- the name of the worker

  `is_enabled`   BOOLEAN            NOT NULL ,      -- is enabled for replication
  `is_read_only` BOOLEAN            NOT NULL ,      -- a subclass of 'is_enabled' which restricts use of
                                                    -- the worker for reading replicas. No new replicas can't be
                                                    -- placed onto this class of workers.

  `svc_host`     VARCHAR(255)       NOT NULL ,      -- the host name on which the worker server runs
  `svc_port`     SMALLINT UNSIGNED  DEFAULT NULL ,  -- override for the global default

  `fs_host`      VARCHAR(255)       NOT NULL ,      -- the host name on which the built-in FileServer runs
  `fs_port`      SMALLINT UNSIGNED  DEFAULT NULL ,  -- override for the global default

  `data_dir`     VARCHAR(255)       DEFAULT NULL ,  -- a file system path to the databases

  -- Connection parameters for the Qserv worker database

  `db_host`      VARCHAR(255)       NOT NULL ,      -- the host name on which the worker database server runs
  `db_port`      SMALLINT UNSIGNED  DEFAULT NULL ,  -- override for the global default
  `db_user`      VARCHAR(255)       DEFAULT NULL ,  -- override for the global default
  `db_password`  VARCHAR(255)       DEFAULT NULL ,  -- override for the global default

  PRIMARY KEY (`name`) ,

  UNIQUE  KEY (`svc_host`, `svc_port`) ,
  UNIQUE  KEY (`fs_host`,  `fs_port`) ,
  UNIQUE  KEY (`db_host`,  `db_port`)
)
ENGINE = InnoDB;
```
The default value (the 'constant') which might address this could be defined either via a common default here:
```
...
INSERT INTO `config` VALUES ('worker', 'svc_port',                   '25000');
INSERT INTO `config` VALUES ('worker', 'fs_port',                    '25001');
INSERT INTO `config` VALUES ('worker', 'data_dir',                   '/qserv/data/mysql');
INSERT INTO `config` VALUES ('worker', 'db_port',                    '3306');
INSERT INTO `config` VALUES ('worker', 'db_user',                    'root');
...
```
Or, perhaps in the above mentioned table schema:
```
  `db_host`      VARCHAR(255)       DEFAULT '127.0.0.1. ,      -- the host name on which the worker database server runs
```

Reply:
Though, I tried to avoid this kind of defaults in the general DDL schema files.